template/11-monetization.md

```diff
@@ -1,14 +1,18 @@
 # Business Model
 
-*Expected operating Costs*
+## Revenue Streams
 
-- Development costs  
-- Infrastructure costs: hosting, storage, and bandwidth  
-- Maintenance costs  
-- Marketing costs  
-- Operational costs: customer Support, administration, legal and compliance  
+Our business model for Proxima is mainly based on advertising for businesses of any sizes. Our main value proposition is providing localized-based content, allowing businesses of any size to reach their targeted audience. This enables a campaign of any size based on the live location of the user, offering from worldwide coverage to precise targeting down to districts or even neighborhoods for centralized actions. This can be viewed as a digital version of local billboards.
 
-*Revenue Streams*
+Our localized based content is also able to provide event organizers (such as festival organizers) time-limited event experiences for users within certain predefined areas. This is aimed to be beneficial to the three parties interacting. Our value proposition is to provide participant retention and event discovery through our heat map. We can engage the participants through challenges, point-earning and enlarged post reach during the event. This fosters application discovery and user loyalty.
```

Comment:
```suggestion
Our location based content is also able to provide event organizers (such as festival organizers) time-limited event experiences for users within certain predefined areas. This is aimed to be beneficial to the three parties interacting. Our value proposition is to provide participant retention and event discovery through our heat map. We can engage the participants through challenges, point-earning and enlarged post reach during the event. This fosters application discovery and user loyalty.
```

Reply:
Thank you ! 
Fixed in: [a450dcf](https://github.com/ProximaEPFL/prd/pull/6/commits/a450dcfd0e2a7c455a45da2072db2395d93fde0b)